README.md

```diff
@@ -1596,6 +1596,7 @@ If you're using [Antigen](https://github.com/zsh-users/antigen), you can test th
 - [avit-mod](https://github.com/zlsun/avit-mod) - Modified version of oh-my-zsh's [avit](https://github.com/ohmyzsh/ohmyzsh/blob/master/themes/avit.zsh-theme) theme.
 - [avoleo](https://github.com/flameleo11/avoleo-zsh-theme) - Features a date and time prompt for each command, as well as a command number in history. In addition, it uses special symbols '⠾' and '⡶' to display `git` information if applicable in the current path. It also supports custom colors based on the Gnome-Terminal default color palette.
 - [aws](https://github.com/chiemerieezechukwu/aws-zsh-theme) - Based on [robbyrussell](https://github.com/ohmyzsh/ohmyzsh/wiki/Themes#robbyrussell), with an extra decorator to show your `$AWS_PROFILE` when it is set.
+- [backbone](https://github.com/lmllrjr/backbone-zsh-prompt) - Bare minimum single file prompt, fast as the roadrunner. Includes decorators for current directory, `git`, `hg`, `mq` and `quilt` information.
```

Comment:
Hey @unixorn 👋,
first of all thank you 🙌. There is just a small thing i want to mention here:

I never intended to support `hg`, `mq` and `quilt` at all as i don't use any of them them 🙈. Moreover i am about to remove all code related to setting up those "version control systems" (There is only a little bit of code for `hg` mercurial left that is not used).
To have an even more reduced prompt that just supports `git` as a version control system. Therefore i kindly ask you to remove the unsupported vcs: ... `hg`, `mq` and `quilt` 🙏. I do not want to mislead anyone here to something that might never exist.

```console
Includes decorators for current directory and `git` information.
```

Sorry that the code of the prompt might was/is still misleading there.